py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -812,3 +812,34 @@ def aten_ops_isinf(
         name,
         args[0],
     )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.convolution.default)
+def aten_ops_convolution(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    # we do not handle transposed.
+    if args[6] is True:
+        raise RuntimeError(f"Target {target} does not support `transposed=True` ")
+    # we do not handle output_padding.
+    if args[7] not in ([0], [0, 0], [0, 0, 0]):
+        raise RuntimeError(f"Target {target} has non-0 output_padding")
```

Comment:
Could these be raised into a capability validator, so they don't throw errors? For instance, you could add:
```python
def convolution_check(convolution_node: Node) -> bool:
    return (not args[6]) and (args[7] in ([0], [0, 0], [0, 0, 0]))

@dynamo_tensorrt_converter(torch.ops.aten.convolution.default, capability_validator=convolution_check)
```